packages/2018/src/tags.js

```diff
@@ -1,5 +1,25 @@
 export default {
-  topics: ["Transportation", "Disaster Resilience", "Housing", "Race"],
-  locations: ["Portland", "Oregon", "Nationwide", "Your City"],
+  topics: [
+    "Transportation",
+    "Disaster Resilience",
+    "Housing",
+    "Race",
+    "Earthquake",
+    "Homeownership",
+    "Wealth",
+    "Redlining",
+    "HOLC",
+    "Permits",
+    "ADUs",
+    "Black",
+    "Gentrification"
```

Comment:
@jaronheard, do you want these extra tags included or removed?

Reply:
Remove Black, keep the rest